src/target/riscv/batch.h

```diff
@@ -44,6 +44,9 @@ struct riscv_batch {
 	/* The read keys. */
 	size_t *read_keys;
 	size_t read_keys_used;
+
+	/* Flag, indicating that the last run of the batch was successful. */
+	bool was_run;
```

Comment:
Here I'd recommend to avoid the word "successful" because someone could interpret it as "busy not encountered", which is not the true meaning here.

I have tried to reword the comment:

```suggestion
	/* Flag indicating that in the last run of the batch, all requested scans were
	 * executed (regardless of the outcome - busy or non-busy) */
	bool was_run;
```

Reply:
Thanks for the suggestion!
The concern is valid.
However, I would word the comment differently. It's important, that the run didn't encounter an error.
Please, take a look.

Reply:
I'm sorry to keep nit-picking on this comment. The current phrasing ("without an error") still may lead the reader to believe that "DMI busy was not encountered" - because "busy" can be considered as a type of an error.

Another proposal:

```
/* Flag indicating that the last run of the batch finished without an error from the underlying OpenOCD's JTAG layer
 * - all JTAG scans were performed. (However, RISC-V DMI "busy" condition could have still been encountered.)
 */
```

Reply:
Addressed